debian/changelog

```diff
@@ -1,3 +1,96 @@
+flatpak (0.11.3-3endless0) unstable; urgency=medium
```

Comment:
Should be `endless1`

Reply:
Further down there's a `0.10.1-1endless0` release. Was that wrong?

Reply:
Yeah... there's nothing *wrong* in the sense that stuff would break, but I believe that we usually count from 1 for that version.